spa/src/components/portal/ContributionsList/Appeal/Appeal.tsx

```diff
@@ -0,0 +1,57 @@
+import MultipleArrows from 'assets/icons/multiple_arrows.svg';
+import PortalAppealImage from 'assets/images/portal-appeal.jpg';
+import { RevenueProgram } from 'hooks/useContributionPage';
+import PropTypes, { InferProps } from 'prop-types';
+import { AppealButton, Description, Image, MultipleArrowsIcon, TextWrapper, Title, Wrapper } from './Appeal.styled';
+
+const AppealLink = AppealButton as any;
+
+export interface AppealProps extends InferProps<typeof AppealPropTypes> {
+  revenueProgram?: Pick<RevenueProgram, 'contributor_portal_show_appeal' | 'website_url'>;
+  /**
+   * If true, the appeal will render in a way to best fit being inside a modal
+   * @default false
+   * */
+  isInsideModal?: boolean;
+}
+
+const Appeal = ({ isInsideModal, revenueProgram }: AppealProps) => {
+  if (!revenueProgram?.contributor_portal_show_appeal) {
+    return null;
+  }
+
+  return (
+    <Wrapper data-testid="appeal" $isInsideModal={!!isInsideModal}>
+      <MultipleArrowsIcon src={MultipleArrows} $isInsideModal={!!isInsideModal} alt="Arrows pointing up icon" />
```

Comment:
```suggestion
      <MultipleArrowsIcon src={MultipleArrows} $isInsideModal={!!isInsideModal} alt="" />
```
Since it's just decorative, empty alt text is most appropriate.